Source/WebCore/style/StyleTreeResolver.h

```diff
@@ -110,7 +110,10 @@ class TreeResolver {
     };
 
     Scope& scope() { return m_scopeStack.last(); }
+    const Scope& scope() const { return m_scopeStack.last(); }
```

Comment:
It’s not immediately apparent why this change is required, I don’t see `scope()` being used in the patch. Is this just to match the addition of `parent()` returning a `const` value?